From 8e067bfaa3a51d906f51e779cf1b2bf3cd9610dc Mon Sep 17 00:00:00 2001
From: Geoffroy Van Cutsem <geoffroy.vancutsem@intel.com>
Date: Mon, 30 Apr 2018 19:14:21 +0200
Subject: [PATCH] Use Clearlinux standard bootloader

Update the hypervisor to use the standard Clearlinux bootloader.
This replaces the previous mechanism where the user had to make
a copy of the bootloader and call it bootloaderx64_origin.efi.
This is no longer needed and will simplify the instructions for
how to install the ACRN hypervisor with a Clearlinux-based
Service OS

Signed-off-by: Geoffroy Van Cutsem <geoffroy.vancutsem@intel.com>
---
 bsp/uefi/include/bsp/bsp_cfg.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bsp/uefi/include/bsp/bsp_cfg.h b/bsp/uefi/include/bsp/bsp_cfg.h
index 16ec336..53b145c 100644
--- a/bsp/uefi/include/bsp/bsp_cfg.h
+++ b/bsp/uefi/include/bsp/bsp_cfg.h
@@ -48,5 +48,5 @@
 #define	CONFIG_DMAR_PARSE_ENABLED	1
 #define	CONFIG_GPU_SBDF		0x00000010	/* 0000:00:02.0 */
 #define CONFIG_EFI_STUB       1
-#define CONFIG_UEFI_OS_LOADER_NAME  "\\EFI\\org.clearlinux\\bootloaderx64_origin.efi"
+#define CONFIG_UEFI_OS_LOADER_NAME  "\\EFI\\org.clearlinux\\bootloaderx64.efi"
 #endif /* BSP_CFG_H */
